common/client/node_lifecycle.go

```diff
@@ -146,7 +137,7 @@ func (n *node[CHAIN_ID, HEAD, RPC]) aliveLoop() {
 		pollFinalizedHeadCh = pollT.C
 	}
 
-	_, highestReceivedBlockNumber, _ := n.StateAndLatest()
+	localHighestChainInfo, _ := n.rpc.GetInterceptedChainInfo()
```

Comment:
why do we use `GetInterceptedChainInfo` and not `n.StateAndLatest` here?

Reply:
`n.StateAndLatest` require more compute than `GetInterceptedChainInfo`